module/zfs/dbuf.c

```diff
@@ -221,8 +221,8 @@ static boolean_t dbuf_evict_thread_exit;
  * by those caches' matching enum values (from dbuf_cached_state_t).
  */
 typedef struct dbuf_cache {
-	multilist_t cache;
 	zfs_refcount_t size ____cacheline_aligned;
+	multilist_t cache;
```

Comment:
This breaks the whole idea of ____cacheline_aligned, which is to split read-only "cache" field from write-heavy size.  Considering DB_CACHE_MAX is 2, this space saving does not matter.  Would we break the structure into two fields though, we could pack the "cache" fields into one cache line.

Reply:
I see what you mean. I am going to withdraw this proposal since the padding has a purpose here.